### PR TITLE
feat!: update npm dependencies to latest versions

### PR DESCRIPTION
**Pull request by cyck**

### CODE REVIEW

File: .prettierrc.cjs
Comment (celsopalmeiraneto) [inline .prettierrc.cjs:1]: Should we add `prettier` as one of the `devDependencies` of this project?

**Review by celsopalmeiraneto — CHANGES_REQUESTED** [review]

File: .prettierrc.cjs
Comment (cyck) [reply .prettierrc.cjs:1]: Honestly, I don't want to deal with that within this scope. I'm happy to remove the config, tho

Comment (celsopalmeiraneto) [reply .prettierrc.cjs:1]: Fine by me.

Comment (cyck) [reply .prettierrc.cjs:1]: Done 🙇🏼

**Review by celsopalmeiraneto — APPROVED** [review]

**Review by sikozonpc — APPROVED** [review]

**Pull request merged**